docs/content/stable/develop/drivers-orms/nodejs/sequelize.md

```diff
@@ -88,6 +88,29 @@ The following code creates an Employees model to store and retrieve employee inf
 - The actual table is created by calling the `Employee.sync()` API in the `createTableAndInsert()` function. This also inserts the data for three employees into the table using the `Employee.create()` API.
 - Finally, you can retrieve the information of all employees using `Employee.findAll()`.
 
+#### Using the YugabyteDB Smart Driver
+
+You can also use the YugabyteDB smart driver (`@yugabytedb/pg`) which provides connection load balancing feature that spreads connections uniformly across the nodes in the entire cluster or in specific placements (zones or regions).
+
+Read more on the smart driver features [here](../../smart-drivers/#using-yugabytedb-smart-drivers).
+
+To use the YugabyteDB smart driver instead of the vanilla PG driver, add the following to your application's `package.json` dependencies:
```

Comment:
```suggestion
To use the YugabyteDB node-postgres smart driver instead of the vanilla pg driver, add the following dependencies to your application's `package.json` file, to override the default PostgreSQL driver with the YugabyteDB smart driver:
```